contracts/tests/SecuredLine.t.sol

```diff
@@ -364,7 +364,7 @@ contract SecuredLineTest is Test {
 
 // declareInsolvent
     function test_must_be_in_debt_to_go_insolvent() public {
-        vm.expectRevert(ILineOfCredit.NotBorrowing.selector);
+        vm.expectRevert(ILineOfCredit.NotLiquidatable.selector);
```

Comment:
We need a test to check the bug that we are fixing.

We must be able to liquidate or declare insolvent if open ppositions exist but do not have principal/debt